app/assets/javascripts/dradis/plugins/csv/upload.js

```diff
@@ -1,12 +1,38 @@
 window.addEventListener('job-done', function(e){
-  var jobId = e.detail.job_id,
-      uploader = document.getElementById('uploader');
+  if ($('body.upload.index').length) {
```

Comment:
shouldn't this be at the top to avoid hooking a listener for every other page in the app?

Reply:
this is fixed in another branch